docs/_includes/api/extras.html

```diff
@@ -8,6 +8,20 @@
 
 These adapters are alternatives to PouchDB's built-in IndexedDB/WebSQL adapters. They fully pass the PouchDB test suite.
 
+#### FruitDOWN adapter
+
+{% highlight js %}
+var PouchDB = require('pouchdb');
+require('pouchdb/extras/fruitdown');
+
+var db = new PouchDB('mydb', {adapter: 'fruitdown'});
+{% endhighlight %}
+
+PouchDB adapter using [FruitDOWN](https://github.com/nolanlawson/fruitdown).
+This is an alternative to the default IndexedDB adapter that ships with PouchDB, which works over all implementations of IndexedDB, including Apple's buggy version.
```

Comment:
I kinda object to calling it "buggy" here. In my own repo attached to my own name, that's fine, but I would prefer for PouchDB to be a bit more objective.